week0001/fizz_buzz_congpine.rb

```diff
@@ -0,0 +1,11 @@
+for counter in 1..100
+
+  if counter % 3 == 0
+    p "Fizz" 
```

Comment:
Use `puts` not `p` because `p` wraps your string in double quotes and hence produces wrong output!